src/partials/section4.html

```diff
@@ -1 +1,85 @@
+                    <source media=" (min-width: 767px)"
+                        srcset="images/tablet/coffe-nuts-chocolate-t.jpg 1x, images/tablet/coffe-nuts-chocolate-t-2x.jpg 2x"
+                        alt="Coffee and nuts chocolate" width="344" height="464">
+
+                    <!-- Mobile -->
+                    <source
+                        srcset="images/mobile/coffe-nuts-chocolate-m.jpg 1x, images/mobile/coffe-nuts-chocolate-m-2x.jpg 2x"
+                        alt="Coffee and nuts chocolate" width="335" height="464">
+
+                    <img src="images/desktop/coffe-nuts-chocolate-d.jpg" alt="Coffee and nuts chocolate">
+                </picture>
+
+                <p class="top-sellers-text mobile-text">Crazy enough chocolate originates from a bean just like the
+                    coffee
+                    bean
+                    called
+                    Cacao.
+                    Cacao
+                    has been
+                    around for thousands of years and was first discovered by the Native American tribe called the
+                    Mayans.
+                </p>
+            </li>
+
+            <li class="sellers-item">
+                <picture>
+                    <!-- Desktop -->
+                    <source media="(min-width: 1200px)"
+                        srcset="images/desktop/orange-mint-chocolate-d.jpg 1x, images/desktop/orange-mint-chocolate-d-2x.jpg 2x"
+                        alt="Orange and mint chocolate" width="368" height="464">
+
+                    <!-- Tablet -->
+                    <source media=" (min-width: 767px)"
+                        srcset="images/tablet/orange-mint-chocolate-t.jpg 1x, images/tablet/orange-mint-chocolate-t-2x.jpg 2x"
+                        alt="Orange and mint chocolate" width="344" height="464">
+
+                    <!-- Mobile -->
+                    <source
+                        srcset="images/mobile/orange-mint-chocolate-m.jpg 1x, images/mobile/orange-mint-chocolate-m-2x.jpg 2x"
+                        alt="Orange and mint chocolate" width="335" height="464">
+
+                    <img src="images/desktop/orange-mint-chocolate-d.jpg" alt="Orange and mint chocolate">
+                </picture>
+            </li>
+
+            <li class="sellers-item">
+                <picture>
+                    <!-- Desktop -->
+                    <source media="(min-width: 1200px)"
+                        srcset="images/desktop/hazelnuts-dark-chocolate-d.jpg 1x, images/desktop/hazelnuts-dark-chocolate-d-2x.jpg 2x"
+                        alt="Hazelnuts dark chocolate" width="368" height="464">
+
+                    <img src="images/desktop/hazelnuts-dark-chocolate-d.jpg" alt="Hazelnuts dark chocolate">
+                </picture>
+            </li>
+        </ul>
+        <div class="top-sellers-text-box container">
+            <p class="top-sellers-text tablet-text">Crazy enough chocolate originates from a bean just like the coffee
```

Comment:
observ ca ai pus textul de 2 ori, e acelasi atat pe mobil, cat si pe tablet/desktop

Reply:
Textue e de doua ori ptr ca e diferenta intre mobile si tablet de pozitie in pagina

Reply:
Rezolvat cu centrarea

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Ordering_flex_items

totusi dupa parerea mea nu e bine sa fie de doua ori textul, incearca pt mobile sa schimbi ordinea, uita te putin pe link ul pe care l am pus aici, daca tot nu reusim, o sa apelez la Victor pt parere/solutie